apps/preview/next/pages/showcases/Textarea/TextareaCharacters.tsx

```diff
@@ -33,7 +33,7 @@ export default function TextareaWithLimit() {
           invalid={invalid}
           disabled={disabled}
           onChange={onChange}
-          className="w-full mt-0.5"
+          className="w-full mt-0.5 block"
```

Comment:
Question: why is that block here?

Reply:
textarea i inline-block by default so there is unexpected gap below textarea therefor wrong ui